cluster/gce/container-linux/configure-helper.sh

```diff
@@ -266,12 +266,12 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
+    token: ${KUBELET_TOKEN}
 clusters:
 - name: local
   cluster:
     certificate-authority-data: ${KUBELET_CA_CERT}
+    server: https://${KUBERNETES_MASTER_NAME}:443
```

Comment:
well, this line wouldn't need to be conditional, the `--kubeconfig` arg to the kubelet would be conditional